core/trino-spi/pom.xml

```diff
@@ -217,6 +217,33 @@
                                         <match>@java.lang.Deprecated(*) ^*;</match>
                                     </old>
                                 </item>
+                                <!-- Revapi tries to compare any API with its previous version -->
+                                <!-- If it can't download the old version, if falls back to comparing with an empty file -->
```

Comment:
Should that result in a problem bigger than 4 airlift classes?

Reply:
also shouldn't `Disable revapi error triggered in offline mode` come before `Add Maven --offline flag as a default option`?

Reply:
> Should that result in a problem bigger than 4 airlift classes?

I also expected a lot more problems because of this, but it seems to work.